config/config.go

```diff
@@ -246,3 +246,12 @@ func GetEtcdServerAddrs() []string {
 	}
 	return []string{"http://localhost:2379"}
 }
+
+func IsTLSenabled() bool {
```

Comment:
I think ws_use_tls will confuse the user, you don't want tls don't enable the server, BTW with we can't listen normal an tls flow on the same port